problems/lines/problem.md

```diff
@@ -40,13 +40,13 @@ $ echo -e 'one\ntwo\nthree' | node split.js
 'three'
 ```
 
-Your own program could use `split` in this way, and you should transform the
+Your own program could use `split2` in this way, and you should transform the
 input and pipe the output through to `process.stdout`. Keep in mind that,
 if you decide to use this technique, `split2` might be actually needed,
 depending on the versions of the other dependencies.
```

Comment:
we can remove this sentence (`Keep in mind ...`), since we are recommending use `split2` in all problem.